lib/spack/spack/test/cmd/stage.py

```diff
@@ -31,27 +32,30 @@ def fake_stage(pkg, mirror_only=False):
     assert len(expected) == 0
 
 
-def test_stage_path(monkeypatch):
-    """Verify that --path only works with single specs."""
+@pytest.fixture(scope='function')
+def tmp_stage_path(monkeypatch, tmpdir):
```

Comment:
Can you rename this to `check_stage_path`? The tests are hard to read currently because it's unclear where the actual checking is happening, this will make it more apparent. Otherwise LGTM

Reply:
Done! Thank you for thinking about this.